cms/static/js/collections/video.js

```diff
@@ -0,0 +1,55 @@
+define([
```

Comment:
I'd add doc string for this file which tells that this video collection is for previous uploaded videos since we're using general file name.